managed/devops/bin/yb_platform_util.sh

```diff
@@ -0,0 +1,525 @@
+    printf "\t Example: \n\t\tbash yb_platform_util.sh create_universe"
+    printf " -f test-universe.json -n test-universe-by-name"
+    printf "\n\n-y | --yes \n\t Input yes for all confirmation prompts\n"
+    printf "\t Example: \n\t\tbash yb_platform_util.sh delete_universe"
+    printf " -n test-universe-by-name -y"
+    printf "\n\n--no-wait \n\t To run command in background and do not wait for task completion task\n"
+    printf "\t Example: \n\t\tbash yb_platform_util.sh create_universe"
+    printf " -f test-universe.json --no-wait"
+    printf "\n\n-t | --task \n\t Task UUID to get task status\n"
+    printf "\t Example: \n\t\tbash yb_platform_util.sh task_status"
+    printf " -t f33e3c9b-75ab-4c30-80ad-cba85646ea39"
+    printf "\n\n-h | --help \n\t Print help message\n\t Example: "
+    printf "\n\t\t bash yb_platform_util.sh -h\n\n"
+}
+
+USER_DIR=$(pwd)
+BASEDIR=$(dirname "$0")
+cd "$BASEDIR"
+
+if [[ ! -z "$1" ]]
+then
+    case $1 in
+        -h|--help)
+            help="True"
+            shift # argument
+            ;;
+        *)
+            operation=$(echo "$1" | tr '[:lower:]' '[:upper:]')
+            shift # argument
+            ;;
+    esac
+else
+    echo "Error: Invalid Actions."
+    HelpMessage
+    exit
+fi
+# unset customer_uuid
+# unset universe_uuid
+# unset universe_name
+# unset input_file
+# unset help
+# unset confirmation
+# unset skip_wait
+# unset task_id
+
+# Loop to fetch all the values fro the user
+# Example, sh yb_platform_util.sh -c f33e3c9b-75ab-4c30-80ad-cba85646ea39 -n jd-script-8-2-21 -o GET
+# this will set customer id name and opertation as get.
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -c|--customer_uuid)
+            customer_uuid="$2"
+            shift # argument
+            shift # value
+            ;;
+        -u|--universe_uuid)
+            universe_uuid="$2"
+            shift # argument
+            shift # value
+            ;;
+        -n|--universe_name)
+            universe_name="$2"
+            shift # argument
+            shift # value
+            ;;
+        -f|--file)
+            input_file="$USER_DIR/$2"
+            shift # argument
+            shift # value
+            ;;
+        -h|--help)
+            help="True"
+            shift # argument
+            ;;
+        -y|--yes)
+            confirmation="y"
+            shift # argument
+            ;;
+        --no-wait)
+            skip_wait="y"
+            shift # argument
+            ;;
+        -t|--task)
+            task_id="$2"
+            shift # argument
+            shift # value
+            ;;
+        *)
+            echo "Error: Invalid command line argument"
+            HelpMessage
+            exit 255
+            ;;
+    esac
+done
+
+
+# Checking if system has python3 installed.
+# If python3 installed run script using python 3 use 2 otherwise.
+
+python_3="False"
+if [[ "$(python3 -V)" =~ "Python 3" ]]
```

Comment:
Is there really a concern that `python3` would _not_ by python 3?
Wouldn't just a check for `python3` in the $PATH suffice?

Reply:
Change code a little bit. It might be more clear now. I am trying to give priority to python3. The person with having an old laptop will have default python2 and python3 points to python3 that the reason why checking python3. In case Default python is python3 then Python script is taking care of.